src/scss/tab/feature/_covid.scss

```diff
@@ -0,0 +1,17 @@
+.covidContainer {
+    display: flex;
+    align-items: center;
+    width: 100%;
+  }
+  .country{
+    border-radius: 10px;
+    background-color: burlywood;
+    margin:15px;
+    padding:5px;
+  }
+  
+  .flag{
```

Comment:
Rename to "covidFlag" to be more specific.

Reply:
done